### PR TITLE
basepoint for account creation - (Pt. 1)

### PR DESCRIPTION
**Pull request by wetefok**
Ticket: [ATS-21](https://cs375-pos-group-project.atlassian.net/browse/ATS-21)

- Created `createConsumerAccount.html` and `createConsumerAccount.js` under `/public/store/accounts` that has input fields for username, first name, and last name. 
- Created an account table in the database. Table stores:
  - username
  - first_name
  - last_name
  - account_type
- Created an `api/accounts/add` POST API endpoint in `server.js` that will validate info and add a row to the account table in the database.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
         "start": "cd app && node server.js",
         "setup": "psql -U postgres -f setup.sql",
         "setup-demo": "psql -U postgres -f setup.sql && psql -U postgres -f demo-data.sql",
+        "demo-only": "psql -U postgres -f demo-data.sql",
```
Comment (gitnamehere) [inline package.json:6]: I don't think we need this

Comment (wetefok) [reply package.json:6]: i forgot why i added it

Comment (wetefok) [reply package.json]: removed now

**Review by andllrew — APPROVED** [review]
FIREEEEE ACCOUNT CREATIONNNN!!!! I WILL SUGGEST THAT YOU LET THE REVIEWERS KNOW HOW TO TEST IT NEXT TIME BECAUSE I FORGOT TO DO NPM RUN SETUP TO CREATE THE TABLE FOR ACCOUNTS!!! 👽👽👽👽

**Review by gitnamehere — APPROVED** [review]

**Comment by wetefok** [comment]
> FIREEEEE ACCOUNT CREATIONNNN!!!! I WILL SUGGEST THAT YOU LET THE REVIEWERS KNOW HOW TO TEST IT NEXT TIME BECAUSE I FORGOT TO DO NPM RUN SETUP TO CREATE THE TABLE FOR ACCOUNTS!!! 👽👽👽👽

MY BAD GANGAROO I DEF GOT YOU NEXT TIME MY HOMIE

**Pull request merged**